client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss

```diff
@@ -4,44 +4,69 @@
 .intro {
 	&.newsletter {
 		background-color: rgb(162 218 254);
-		background-image:
-			url(calypso/assets/images/onboarding/newsletter-intro-left.png),
-			url(calypso/assets/images/onboarding/newsletter-intro-right.png);
-		background-repeat: no-repeat, no-repeat;
-		background-attachment: fixed;
-		/*!rtl:ignore*/
-		background-position-x: 0%, 100%;
-		background-position-y: 100%, 80px;
-		background-size: auto 30%, auto 50%;
-
-
-		@include break-huge {
-			background-size: auto 55%, auto 80%;
-			background-position-y: 100%, 136px;
+
+		.step-container {
+			&.is-using-new-intro-design {
+				background-image: url(calypso/assets/images/onboarding/newsletter-intro-bottom.png);
```

Comment:
This bloody image is 660KB, would you be able to make a smaller one and use it here, and us this big one only on desktop?

Reply:
For sure 👍
Makes sense to me